src/Config.hs

```diff
@@ -55,6 +55,7 @@ fromIni = do
       (configShowTypingIndicator defaultConfig)
     configEnableAspell <- fieldFlagDef "enableAspell"
       (configEnableAspell defaultConfig)
+    configActivityNotifyCmd <- fieldMb "activityNotifyCmd"
```

Comment:
Please rename the setting name to `activityNotifyCommand` (just the configuration setting string, not the record field name).

Also, can you please add a section on this new setting to `sample-config.ini`?